### DISCUSSION TITLE
DOS 6.22/Windows 3.11 Guest

### DISCUSSION THREAD

**Discussion by harlleysathler** [utmapp/UTM#5156] [UTM Help (macOS)]
Hello all!

I don't know if this was asked before and I'm new to UTM. I searched the forum for Windows 3, but can't find anything.

Yesterday I successfully installed MS-DOS 6.22 and Windows 3.11. What I couldn't do, however, was set the screen resolution higher than 640x480/16 colors, neither network.

However, the network is not critical, as I can put data into the VM using ISO images (although I can't think of a way to get data out of it), but I would like to get the screen a better resolution.

Can anyone tell me the best VM settings and what drivers I can use? I set the video card to the Cirrus Logic GD something and successfully got its drivers installed but when using their utility to change screen resolution, it crashes with a message error (no BSOD).

I set the network card to NE2000 compatible, but if I tried to enable network support in Windows, it can't start.

I really appreciate any help! Thank you all!

**Comment by tifasoftware** [comment]
Are you using i430fx or isapc?

**Reply by harlleysathler** [reply]
i430fx. Should I use  isapc?

**Reply by tifasoftware** [reply]
I recommend trying isapc. Then switch Cirrus to Cirrus-isa and disable QEMU Agent options as they cause issues with isapc. (You wouldn't use QEMU agents with i430fx on 3.11 anyways)

**Reply by harlleysathler** [reply]
Thanks for the tip! Unfortunately, it didn't work. However, I tried to keep the settings with i430fx and set the video card as vmware. In the DOS/Windows 3.11 I used the VGA Driver patch to set the resolution to 1024x760/256. This works. I will try again with isapc and another driver as the chipset detection program that was packaged with Cirrus Logic drivers indicates a different chipset than that of the driver.

**Reply by harlleysathler** [reply]
[UPDATE] - I reverted the video settings to Cirrus CLGD 54xx VGA and tried the driver version in this archive.org link: https://archive.org/details/cirrus-logic-cl-gd-5434-windows-3.1x-drivers

This worked pretty well! I'm now running my Windows 3.11 VM in 1280x1024 with 64 thousand colors. This version installs the utility as well but also lists the drivers in Windows Settings. The utility is not useful, but the drivers listed made all difference.

I'll try to make get network now. Does anyone know if it's possible to setup a guest/host network in this version of Windows?

**Comment by tifasoftware** [comment]
Ne2000 isa should work bestSent from my iPhoneOn Mar 30, 2023, at 6:06 PM, harlleysathler ***@***.***> wrote:﻿
[UPDATE] - I reverted the video settings to Cirrus CLGD 54xx VGA and tried the driver version in this archive.org link: https://archive.org/details/cirrus-logic-cl-gd-5434-windows-3.1x-drivers
This worked pretty well! I'm now running my Windows 3.11 VM in 1280x1024 with 64 thousand colors. This version installs the utility as well but also lists the drivers in Windows Settings. The utility is not useful, but the drivers listed made all difference.
I'll try to make get network now. Does anyone know if it's possible to setup a guest/host network in this version of Windows?
